lib/src/components/servo/service.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/servo/servo.dart';
+import 'package:viam_sdk/src/gen/common/v1/common.pb.dart';
+import 'package:viam_sdk/src/gen/component/servo/v1/servo.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+
+class ServoService extends ServoServiceBase {
+  final ResourceManager _manager;
+
+  ServoService(this._manager);
+
+  Servo _servoFromManager(String name) {
+    try {
+      return _manager.getResource(Servo.getResourceName(name));
+    } catch (e) {
+      // TODO make rescource not found GRPC error
+      print('Exception details:\n $e');
+      rethrow;
```

Comment:
Every time I read a flutter PR I learn a new keyword. Turns out dart has a whopping 61 keywords. Go has 25 which is abnormally low, but I thought rust was on the high end with 51. 😮 

Edit: swift has 71 and C++ has 95 so I don't even know anymore 🤷🏻

Reply:
Wow, I had no idea there were so  many. Wild. Is that including meta annotations? 👀

Reply:
> Is that including meta annotations?

I _think_ so, yeah. I did some very fast googling for all of those keyword counts, so unclear how reputable my sources were 😆 .